source/kraken/maintenance_worker.cpp

```diff
@@ -355,13 +359,28 @@ void MaintenanceWorker::init_rabbitmq() {
     queue_name_task = (boost::format("%s_task") % queue_name).str();
     queue_name_rt = (boost::format("%s_rt") % queue_name).str();
 
-    // connection through URI or classic method
-    if (broker_uri) {
-        LOG4CPLUS_INFO(logger, boost::format("connection to rabbitmq with uri: %s") % *broker_uri);
-        channel = AmqpClient::Channel::CreateFromUri(*broker_uri);
-    } else {
-        LOG4CPLUS_INFO(logger, boost::format("connection to rabbitmq: %s@%s:%s/%s") % username % host % port % vhost);
-        channel = AmqpClient::Channel::Create(host, port, username, password, vhost);
+    if (protocol == "amqp") {
+        // connection through URI or classic method
+        if (broker_uri) {
+            LOG4CPLUS_INFO(logger, boost::format("connection to rabbitmq with uri: %s") % *broker_uri);
```

Comment:
is it possible to change the log level or delete the mpd

Reply:
Not relevant anymore, I used the new API from `SimpleAmqpClient` and am able to log everything always, except the password. I believe this should be OK now.